.github/pull_request_template.md

```diff
@@ -7,17 +7,24 @@
 <!-- Enter your proof that it works here.-->
 
 ## Checklist
+
 - [ ] Have you linked a jira ticket and/or is the ticket in the title?
 - [ ] Have you checked whether your jira ticket required DOCSP changes?
-- [ ] Have you checked for release_note changes?
+- [ ] Have you added changelog file?
+    - use `skip-changelog` label if not needed
+    - refer
+      to [Changelog files and Release Notes](https://github.com/mongodb/mongodb-kubernetes/blob/master/CONTRIBUTING.md#changelog-files-and-release-notes)
+      section in CONTRIBUTING.md for more details
```

Comment:
Raising a PR and then for every PR adding `skip-changelog` seems counter intuitive to me. Do we generally require all PRs to have change logs?
I think I am a little unsure about our definition of changelog and release note. If you can clarify that, maybe my question would be answered.